---
title: "C28163 | Microsoft Docs"
ms.custom: ""
ms.date: "11/04/2016"
ms.reviewer: ""
ms.suite: ""
ms.technology: 
  - "vs-devops-test"
ms.tgt_pltfrm: ""
ms.topic: "article"
f1_keywords: 
  - "C28163"
helpviewer_keywords: 
  - "C28163"
ms.assetid: 24fecbde-1c96-4a45-82f7-9f47cfc0ef11
caps.latest.revision: 3
author: "corob-msft"
ms.author: "corob"
manager: "ghogen"
caps.handback.revision: 3
---
# C28163
[!INCLUDE[vs2017banner](../code-quality/includes/vs2017banner.md)]

Warnung C28163: Die Funktion sollte nie innerhalb eines try\/except\-Blocks aufgerufen werden.  
  
 Diese Warnung wird ausgegeben, wenn eine Funktion von einem Typ ist, der nicht in einem `try/except` \-Block enthalten soll wird gefunden in einem `try/except`\-Block.  Das Codeanalysetool konnte mindestens einen Pfad, in den die aufgerufene Funktion innerhalb eines Blocks `try/except` war.